apps/server/src/app/users/users.controller.ts

```diff
@@ -8,27 +8,25 @@ import {
   Get,
 } from '@nestjs/common';
 import { UsersService } from './users.service';
-import { CreateUser } from './dto/create-user.dto';
-import { UpdateUser } from './dto/update-user.dto';
-import { UserLogin } from './dto/user-login.dto';
+import { User } from './dto/user.dto';
 import { Users } from './entities/users.entity';
 
 @Controller()
 export class UsersController {
   constructor(private usersService: UsersService) {}
 
   @Post('signup')
-  signUp(@Body() userParams: CreateUser, @Query('adminId') adminId?: string) {
-    return this.usersService.signup(userParams, adminId);
+  signUp(@Body() userParams: User) {
```

Comment:
```suggestion
  signUp(@Body() userDto: UserDto) {
```
I think nest actually recommends this naming convention